src/DotVVM.Samples.Common/CommonConfiguration.cs

```diff
@@ -16,6 +20,12 @@ public static void ConfigureServices(IServiceCollection services)
             });
 
             services.Configure<DotvvmResourceRepository>(RegisterResources);
+
+            services.Configure<JavascriptTranslatorConfiguration>(c => {
+                c.MethodCollection.AddMethodTranslator(typeof(JavaScriptUtils),
+                   nameof(JavaScriptUtils.LimitLength),
+                   new GenericMethodCompiler((a) => new JsIdentifierExpression("limitLength").Invoke(a)));
```

Comment:
If we'd like to add this function to framework, it does not seem like a good idea to use the limitLength as a global. And even here, it scares me a bit :P

Reply:
I created it only for testing purpose obviously. I can put it in a js object if you want to.